appveyor.yml

```diff
@@ -1,19 +1,21 @@
-version: "{build}"
-configuration: Debug
+version: '{build}'
+image: Visual Studio 2015
+configuration: Release
+install:
+  - ps: mkdir -Force ".\build\" | Out-Null
+  - ps: Invoke-WebRequest "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1" -OutFile ".\build\installcli.ps1"
```

Comment:
https://github.com/serilog/serilog-enrichers-thread is a good starting point I think